relay-metrics/src/aggregation.rs

```diff
@@ -746,6 +748,16 @@ pub struct AggregatorConfig {
     /// before sending such a backdated bucket to the upsteam. This should be lower than
     /// `initial_delay`.
     pub debounce_delay: u64,
+
+    /// The age in seconds of the oldest allowed bucket timestamp.
+    ///
+    /// Defaults to 30 days.
+    pub max_age: u64,
+
+    /// The time in seconds that a timestamp may be in the future.
+    ///
+    /// Defaults to 1 minute.
+    pub max_future: u64,
```

Comment:
I used the same defaults as for events, not sure if that makes sense (@jan-auer).

https://github.com/getsentry/relay/blob/5e22be35a9bba8b686383badc2d55c65eec9f86f/relay-config/src/config.rs#L743-L749